src/Definition/Redshift.php

```diff
@@ -58,9 +58,14 @@ public function getCompression()
     public function getSQLDefinition()
     {
         $definition =  $this->getType();
-        if ($this->getLength() && $this->getLength() != "") {
-            $definition .= "(" . $this->getLength() . ")";
+
+        $typesWithoutLength = ['TIMESTAMP', 'DATE', 'FLOAT', 'BOOLEAN'];
+        if (!in_array($this->getBasetype(), $typesWithoutLength)) {
```

Comment:
Me prijde, ze to co vraci DB by melo jit prave v +- "surove" podobe pouzit v tomhle baliku.
Davat tam nejakou mezivrstvu, ktera odstrani z x datovych typu delku mi prijde zvlastni.